### DISCUSSION TITLE
Why doesnt `useFieldArray` get the values from `useForm`'s `values` prop on mount?

### DISCUSSION THREAD

**Discussion by paulleonartcalvo** [react-hook-form/react-hook-form#12137] [Q&A]
I was using `useForm` and `useFieldArray` like in the documentation, and when i get form `values` from my props, all the Controllers get their correct values but `useFieldArray()`'s fields are empty even though the array is clearly populated in the form values. Is this behavior intended?

My use case is editing a previously submitted form. So i fetch the data from an api then render a Form component and pass in the `values`. I expected the `fields` property to be initially populated with the values from that prop.

**Comment by relkaddari** [comment]
i have the same issue
